packages/storage-ui/src/Components/Pages/BucketsPage.tsx

```diff
@@ -121,16 +121,21 @@ const useStyles = makeStyles(({ breakpoints, animation, constants, typography }:
   })
 )
 
+type SortColumn = "name" | "file_system" | "size" | undefined
```

Comment:
undefined is already an option in the sortColumn state
```suggestion
type SortColumn = "name" | "file_system" | "size"
```